entrypoint.sh

```diff
@@ -11,19 +11,6 @@ done
 
 echo 'DB started'
 
-# It's okay to start Celery in the background and continue without waiting, even though "migrate"
-# might make DB changes we want to notify for since tasks are queued by Django Webhook and
-# are executed as soon as celery starts
-# FIXME: This makes testing locally a bit awkward, since this isn't started by "manage.py runserver"
-#  maybe there's a way to do this better?
```

Comment:
How does this comment change with these changes? Is the guidance still to run these commands from the readme for local dev?
```
celery -A meshdb worker -l INFO
celery -A meshdb beat -s /tmp/celerybeat-schedule -l DEBUG
```

Also does the celery worker we spin up programmatically during unit testing need to be changed in any way?

Reply:
Shouldn't be. This just moves the worker and beat out of the prod container. Grandbox will change, but whatever. I guess what I'd do if I were testing this locally would be to run the provided scripts.

Reply:
Could you update the readme with the latest guidance?

Reply:
Fuck, I knew you were gonna say that.

Reply:
Actually, on second thought, the readme instructions are fine. What's there right now will still work.